Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,91 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
-
-        public static int FindDigit(string equation)
-        {
+	// To find and replace the digit and then to check whether it is valid or not
+	public static int findMissing(int number,int position, String[] nums)

```

Comment:
What is the value of passing `position` and 'nums[]` when you could have simply passed num[position] directly?